### PR TITLE
[FIX] mail: allow accurate translation by providing full context

### PR DESCRIPTION
**Pull request by oomsveta**

### CODE REVIEW

**Comment by alexkuhn** [comment]
@robodoo override=ci/security

Template is mostly static except the thread name, which is escaped as it was before.

**Review by alexkuhn — APPROVED** [review]

**Comment by oomsveta** [comment]
robodoo r+

File: addons/mail/static/src/web/chatter.js [hunk 1]
```diff
@@ -177,6 +179,24 @@ export class Chatter extends Component {
         return this.state.thread?.attachments ?? [];
     }
 
+    /**
+     * @returns {string}
+     */
+    get toFollowersText() {
+        const threadName = this.state.thread.displayName || this.state.thread.name;
+        const toFollowersText = threadName
```
Comment (seb-odoo) [inline addons/mail/static/src/web/chatter.js:187]: markup() of threadName would be a security issue

Comment (seb-odoo) [reply addons/mail/static/src/web/chatter.js:187]: forget it, it's `? :` and not `||`

**Pull request closed without merging**